### PR TITLE
Apply branch-0.2 documentation updates to gh-pages

### PR DESCRIPTION
**Pull request by sameerz**
Signed-off-by: Sameer Raheja <sraheja@nvidia.com>

1. Please write a description in this text box of the changes that are being
   made.

Apply changes from branch-0.2 commit b89d9bd5bf9a9f30490d33d64cbd5bc0101a1a38 to gh-pages.  Related PR #422 

2. Please ensure that you have written units tests for the changes made/features
   added.

Tested locally with `docker run --rm -it  --volume="$PWD:/srv/jekyll" -p 4000:4000 sameerz/jekyll:0.1 jekyll serve`.  Note that in my testing, links with `filename.md#anchor` do not work, but they appear to work correctly on gh-pages.  Related: PR #419 .

### CODE REVIEW

**Comment by revans2** [comment]
build

**Comment by revans2** [comment]
@sameerz this is kind of a side question.  What are we supposed to do about the build for documentation changes.  This branch has no jenkins scripts for premerge so the "build" will always fail. I also don't know what if anything triggers "Page Build" and "github/pages"

**Comment by jlowe** [comment]
I've turned off the two CI checks on this branch until we have something setup to satisfy them.

File: docs/get-started/getting-started.md [hunk 1]
```diff
@@ -373,11 +375,107 @@ operation “count at ...”, you should see the graph of Spark Execs and some o
...
+
+2. Install [UCX 1.8.1](https://github.com/openucx/ucx/releases/tag/v1.8.1). 
+
+
+3. You will need to configure your spark job with extra settings for UCX (we are looking to 
+simplify these settings in the near future). Choose the version of the shuffle manager
+that matches your Spark version. Currently we support
+   - Spark 3.0.0 (com.nvidia.spark.rapids.spark300.RapidsShuffleManager) 
+   - Spark 3.0.1 (com.nvidia.spark.rapids.spark301.RapidsShuffleManager) 
+   - Spark 3.1.0 (com.nvidia.spark.rapids.spark310.RapidsShuffleManager)
```
Comment (jlowe) [inline docs/get-started/getting-started.md:427]: This is talking about shim features which don't exist in 0.1.  We should not update our main doc pages to discuss anything that isn't in the stable release.

Comment (sameerz) [reply docs/get-started/getting-started.md]: Pushed a change to remove 0.2 related documentation.

**Comment by sameerz** [comment]
@revans2 I am not yet familiar with the way the gh-page builder works, or how it reports results.  It is something I need to look into.  

> @sameerz this is kind of a side question. What are we supposed to do about the build for documentation changes. This branch has no jenkins scripts for premerge so the "build" will always fail. I also don't know what if anything triggers "Page Build" and "github/pages"

**Pull request merged**